composer.json

```diff
@@ -36,19 +36,18 @@
         "doctrine/cache": "~1.1",
         "doctrine/orm": "~2.3",
         "enqueue/enqueue-bundle": "^0.7",
```

Comment:
IIRC for versions below 1.0, we need to explicitly allow minor versions .. so I think this needs to be “^0.7|^0.8” in order to be able to pull in the 0.89 version which provides Symfony compatibility

Reply:
fixed via https://github.com/liip/LiipImagineBundle/pull/1013/commits/c5e26d6ac908fc96f801ca070415b648b66e694f